assets/utils/commands/bump.js

```diff
@@ -0,0 +1,163 @@
+  await input.init(true);
+
+  const lib = detectLibrary(input);
+  if (!lib) return;
+
+  if (!runH5pBump(lib)) return;
+
+  process.chdir(lib);
+  output.printLn(`${c.blue}Changed directory back to: ${c.emphasize}${lib}${c.default}`);
+
+  const version = getVersion();
+  if (!version) return;
+
+  if (!stageChanges(autoYes)) return;
+  if (!commitChanges(version)) return;
+
+  if (autoYes) {
+    tagAndPush(version, true, true);
+  } else {
+    promptTagAndPush(version);
+  }
+}
+
+// Helpers
+function detectLibrary(input) {
+  let libraries = input.getLibraries();
+  if (!libraries.length) {
+    const cwd = process.cwd();
+    const libName = path.basename(cwd);
+    libraries = [libName];
+    process.chdir(path.dirname(cwd));
+    output.printLn(`${c.blue}Defaulting to current folder: ${c.emphasize}${libName}${c.default}`);
+  }
+  output.printLn(`${c.blue}Bumping patch version of: ${c.emphasize}${libraries[0]}${c.default}`);
+  return libraries[0];
+}
+
+function runH5pBump(lib) {
+  let bumpOutput = '';
+  try {
+    bumpOutput = execSync(`h5p utils increase-patch-version ${lib}`, { encoding: 'utf8' });
+    output.printLn(bumpOutput.trim());
+  } catch {
+    output.printLn(`${c.red}Failed to bump version using h5p utils.${c.default}`);
+    return false;
+  }
+  if (bumpOutput.includes('SKIPPED')) {
+    output.printLn(`${c.yellow}Nothing to bump — skipping further steps.${c.default}`);
+    return false;
+  }
+  return true;
+}
+
+function getVersion() {
+  const libData = repository.getLibraryData('.');
+  const { majorVersion, minorVersion, patchVersion } = libData;
+  if (!majorVersion || !minorVersion || !patchVersion) {
+    output.printLn(`${c.red}Failed to read version from library.json.${c.default}`);
+    return null;
+  }
+  const version = `${majorVersion}.${minorVersion}.${patchVersion}`;
+  output.printLn(`${c.green}New version: ${version}${c.default}`);
+  return version;
+}
+
+function stageChanges(autoYes) {
+  try {
+    execSync(`git restore --staged library.json`, { stdio: 'inherit' });
+    if (autoYes) {
+      output.printLn(`${c.yellow}Staging version bump automatically…${c.default}`);
+      execSync(`git add library.json`, { stdio: 'inherit' });
+    } else {
+      output.printLn(`${c.yellow}Staging version bump interactively…${c.default}`);
+      execSync(`git add -p library.json`, { stdio: 'inherit' });
+    }
+  } catch {
+    output.printLn(`${c.red}Git staging failed.${c.default}`);
+    return false;
+  }
+
+  const gitStatus = execSync(`git status --porcelain library.json`, { encoding: 'utf8' }).trim();
+  if (!gitStatus) {
+    output.printLn(`${c.yellow}No changes detected in library.json — nothing to commit.${c.default}`);
+    return false;
+  }
+
+  return true;
+}
+
+function commitChanges(version) {
+  try {
+    execSync(`git commit -m "Bump to ${version}"`, { stdio: 'inherit' });
+    output.printLn(`${c.green}Committed version bump to ${version}.${c.default}`);
+    return true;
```

Comment:
Matter of taste only: I'd have expected the `return` statement after the try-catch block, as I expect the "default" `return` statement of a function at the end - here there's none.